docs/user/administration/settings/components.md

```diff
@@ -0,0 +1,14 @@
+# BuddyPress Components settings
+
+The BuddyPress Core & Members components are required ones and are always enabled. The first time you activate ByddyPress, 4 optional components are activated for you, by default: 
```

Comment:
BuddyPress is misspelled here.

Reply:
Hi, thanks for your check, I will update the PR asap 👌